src/Symfony/Component/ErrorRenderer/Resources/views/logs.html.php

```diff
@@ -16,7 +16,11 @@
         } elseif ($log['priority'] >= 300) {
             $status = 'warning';
         } else {
-            $severity = $log['context']['exception']['severity'] ?? false;
+            if (($exception = $log['context']['exception'] ?? null) && $exception instanceof ErrorException) {
+                $severity = $log['context']['exception']->getSeverity();
```

Comment:
```suggestion
                $severity = $exception->getSeverity();
```
Use the variable, otherwise simplifies the `if` statement to:
```php
if (($log['context']['exception'] ?? null) instanceof \ErrorException) {
```

Reply:
fixed it :)

 I just commited it pretty fast after I found this, simplifying part just slipped my mind